docs/concepts.md

```diff
@@ -21,7 +21,8 @@ the frontend requests that all regions that do not intersect with the selected t
 
 # Sources and sourcemapping
 
-When a recording is opened, the frontend loads all javascript sources in that recording using [`Debugger.findSources`](https://static.replay.io/protocol/tot/Debugger/#method-findSources) and stores them in the [`ThreadFront`](../src/protocol/thread/thread.ts). Sourcemaps are handled in the backend, so the sources also include original sources from sourcemaps. Furthermore, the backend automatically adds pretty-printed versions of minified sources. The [`newSource`](https://static.replay.io/protocol/tot/Debugger/#event-newSource) events from the backend include the list of generated sources for every original source. This information is used to pick the sources that are shown to the user: Replay always tries to show an original source and may offer a generated source as an alternate that the user can switch to using the toggle at the bottom of the editor.
+When a recording is opened, the frontend loads all javascript sources in that recording using [`Debugger.findSources`](https://static.replay.io/protocol/tot/Debugger/#method-findSources) and stores them in the [`ThreadFront`](../src/protocol/thread/thread.ts). Sourcemaps are handled in the backend, so the sources also include original sources from sourcemaps. Furthermore, the backend automatically adds pretty-printed versions of minified sources and extracted inline scripts from html pages. The [`SourceKind`](https://static.replay.io/protocol/tot/Debugger/#type-SourceKind) specifies where a source came from.
+The [`newSource`](https://static.replay.io/protocol/tot/Debugger/#event-newSource) events from the backend include the list of generated sources for every original source. This information is used to pick the sources that are shown to the user: Replay always tries to show an original source and may offer a generated source as an alternate that the user can switch to using the toggle at the bottom of the editor.
```

Comment:
> events from the backend include the list of generated sources for every original source

Did you mean: events from the backend include the list of original sources for every generated source?

Reply:
No, see https://static.replay.io/protocol/tot/Debugger/#event-newSource. The `ThreadFront` creates the list of original sources for every generated one from that.

Reply:
Nope, they include a list of generated sources for original sources (at least that's how it works per the docs at https://static.replay.io/protocol/tot/Debugger/#event-newSource)